pkg/config/hardfork.go

```diff
@@ -10,14 +10,15 @@ const (
 	// https://github.com/neo-project/neo/pull/2712) and #2519 (ported from
 	// https://github.com/neo-project/neo/pull/2749).
 	HFAspidochelone Hardfork = 1 << iota // Aspidochelone
+	HFBasilisk                           // Basilisk
```

Comment:
`HFAspidochelone` has some nice comment, I'd expect `HFBasilisk` to have it as well.

Reply:
Fixed. The comment will be extended in the related PRs.